data/xml/2021.naacl.xml

```diff
@@ -34,6 +34,7 @@
       <doi>10.18653/v1/2021.naacl-main.1</doi>
       <bibkey>zhang-etal-2021-knowledge</bibkey>
       <video href="2021.naacl-main.1.mp4"/>
+      <video href="2021.naacl-main.1.mp4"/>
     </paper>
```

Comment:
This is a duplicate tag—it seems many of NAACL are. Can you remove just the duplicates? I assume some are new.

Reply:
Good catch. It turned out that `bin/ingest_archive_video.py` adds duplicate videos if it is used multiple times for the same folder. I updated the script to prevent this in the future. Turns out that there are actually no missing videos for NAACL 2021 main conference; they were all duplicates.

I'll check if the duplicate video issue also affected other pull requests.